fip-0026.md

```diff
@@ -339,251 +409,281 @@ This sets the marketplace configurations.
-|Insufficient Funds|`buyer` doesn't have enough funds|400| | |"Not enough FIO"|
-|Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
-
-#### Response
-TBD
-
-##### Example
-TBD
-
-### Check Expired Listings
-
-#### New action: *chkexplistng*
-This action needs to be run periodically to check the listings to see if they are expired. If they are and the seller has a `notif_address` set, a FIO request for a small amount of FIO will be sent with a message notifying them their listing is about to expire. If there is no `notif_address` and the domain expires, the cancel listing action will be called which will remove the listing from the `domainsale` table and transfer the domain back to the seller.
-
-Anyone can call this action. It only needs to be called once a day, though 
-
-#### Processing
-- Iterate through all domains listed on `domainsale` table. 
-- If `expiration` is less than `warn_dur` days away and there is a `notif_address` on the listing
-    - If `warned_expire` is not true
-    - A notification will be sent to the `notif_address` via a FIO request about the pending expiration
-    - Flip the `warned_expire` bool to true
-- if no `notif_address` provided nothing will happen
-- if now() is greater than `expiration` then the *cxlistdomain* action is called which will transfer the domain back to the seller and delete the listing.
+- Verify the actor is the owner of the marketplace
+- Toggle the e_break field to either 1 or 0, whichever the opposite of what the current value is.
 
 #### Exception handling
 |Error condition|Trigger|Type|fields:name|fields:value|Error message|
 |---|---|---|---|---|---|
-|Invalid marketplace|Supplied `marketplace` is not found in marketplace config table `mrkplconfig`|400| |"Marketplace not found"|
-|Invalid FIO domain|Supplied `fio_domain` is not listed for sale.|400| | |"FIO domain not found"|
-|Insufficient Funds|`buyer` doesn't have enough funds|400| | |"Not enough FIO"|
+|Not owner of marketplace|The signer does not own the marketplace|403| | | |
+|Fee exceeds maximum|Actual fee is greater than supplied max_fee|400|"max_fee"|Value sent in, e.g. "1000000000"|"Fee exceeds supplied maximum"|
+|Insufficient funds to cover fee|Account does not have enough funds to cover fee|400|"max_fee"|Value sent in, e.g. "1000000000"|"Insufficient funds to cover fee"|
+|Invalid TPID|tpid format is not valid|400|"tpid"|Value sent in, e.g. "notvalidfioaddress"|"TPID must be empty or valid FIO address"|
 |Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
 
-#### Response
-TBD
+#### Response body
+|Parameter|Format|Definition|
+|---|---|---|
+|status|String|OK if successful|
+|new_listing_fee|int|New listing fee amount, in SUFs| 
+|fee_collected|Int|Amount of SUFs collected as fee|
 
 ##### Example
-TBD
+```
+{
+  "status": "OK",
+  "fee_collected": 2000000000
+}
+```
 
-### Changes to existing actions
+## Modified actions
```

Comment:
Need to define somewhere:
* What happens if I try to transfer a domain that is listed, presumable error needs to be added.
* Are all other actions allowed? Like registering address, setting domain public, etc.